_data/history.yaml

```diff
@@ -0,0 +1,56 @@
+# Chronological history of the Nottingham New Theatre
+
+- year: 1927
+  title: Formation of Dramsoc
+  description: "The University College Nottingham Dramatic Society (Dramsoc) performed its first play in 1928: A.A. Milne's *Mr Pim Passes By*. <br /> The society offered members an opportunity to read, discuss, produce and enact plays."
```

Comment:
Single BRs in paragraph text are not what we want. You could do multiline yaml then use the `| markdownify` filter to get paragraph tags.

Multiline strings: https://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines

Disclaimer:

> There are ~~5 6~~ NINE different ways to write multi-line strings in YAML

I've used the `>` method in the past.